iron-iconset-svg.html

```diff
@@ -66,8 +66,25 @@
       size: {
         type: Number,
         value: 24
+      },
+
+      /**
+       * Set to true to enable mirroring of icons where specified when they are
+       * stamped. Icons that should be mirrored should be decorated with a
+       * `mirror-in-rtl` attribute.
+       */
+      rtlMirroring: {
```

Comment:
what if it's also just `mirror-in-rtl`? Mehhhhh I don't really have a strong opinion